build/barys

```diff
@@ -220,15 +220,15 @@ while [[ $# -ge 1 ]]; do
         --shared-downloads)
             # Argument needs to be non-empty and absolute path
             if [ -z "$2" ] || [ "${2:0:1}" != "/" ]; then
-                log ERROR "\"$1\" argument is invalid."
+                log ERROR "\"$1\" argument is invalid. Argument needs to be non-empty and absolute path."
```

Comment:
I would reword this as `Argument needs to be a non-empty, absolute path` or as `Argument needs to be non-empty and an absolute path.`